uberfire-backend/uberfire-backend-server/src/test/java/org/uberfire/backend/server/util/PathsTest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.uberfire.backend.server.util;
+
+import org.junit.Test;
+import org.uberfire.backend.vfs.Path;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class PathsTest {
+
+    @Test
+    public void getFilePathFromProjectRoot() throws Exception {
+
+        final Path repositoryRoot = getPath("file:///spaceName/repositoryName",
+                                            "/");
+        final Path pathToFile = getPath("file:///spaceName/repositoryName/org/test/file.txt",
+                                        "file.txt");
+
+        assertEquals("/org/test/file.txt", Paths.removePrefix(pathToFile,
+                                                              repositoryRoot));
```

Comment:
@Rikkola are you sure the result is expected to contain leading slash? Since after removing any prefix the result becomes a relative path, I wouldn't expect it there. For example, in kie-wb-common it is used like this (after applying this PR):
```java
private boolean isIncrementalBuildPossible(final Path resourcePath) throws NoModuleException {
    return getDestinationPath(resourcePath).startsWith("src/main/resources/");
}
private String getDestinationPath(final Path originalPath) throws NoModuleException {
    return Paths.removePrefix(originalPath, module(originalPath).getRootPath());
}
```
So, will this work? Is it guaranteed that after removing module root from `resourcePath` it will start with `src/main/resources` and not `/src/main/resources`?